stdlib/2and3/shutil.pyi

```diff
@@ -96,12 +96,8 @@ else:
                                         Iterable[AnyStr]]] = ...) -> _PathReturn: ...
 
 if sys.version_info >= (3,):
-    @overload
-    def rmtree(path: bytes, ignore_errors: bool = ...,
-               onerror: Optional[Callable[[Any, str, Any], Any]] = ...) -> None: ...
-    @overload
-    def rmtree(path: _AnyPath, ignore_errors: bool = ...,
-               onerror: Optional[Callable[[Any, _AnyPath, Any], Any]] = ...) -> None: ...
+    def rmtree(path: Union[str, _Path], ignore_errors: bool = ...,
```

Comment:
```suggestion
    def rmtree(path: Union[bytes, _Path], ignore_errors: bool = ...,
```

Reply:
Thank you! I should pay more attention to what I'm typing.